### PR TITLE
JBIDE-27865 - add wfly24 with modifications to allow reuse

### PR DESCRIPTION
**Pull request by robstryker**

### CODE REVIEW

File: as/plugins/org.jboss.ide.eclipse.as.core/plugin.xml [hunk 1]
```diff
@@ -3020,5 +3020,77 @@
...
+      <runtime-component-version
+            type="org.jboss.ide.eclipse.as.runtime.wildfly.component"
+            version="24.0"/>
+     <supported>
+        <runtime-component
+         	id="org.jboss.ide.eclipse.as.runtime.wildfly.component"
+         	version="24.0"/>
+       <facet id="wst.web"  version="1.0,1.2,1.3,1.4"/>
+       <facet id="jst.web"  version="2.2,2.3,2.4,2.5,3.0,3.1,4.0"/>
+       <facet id="jst.java" version="5.0,6.0,1.7,1.8,9,10,11,12"/>
```
Comment (jeffmaury) [inline as/plugins/org.jboss.ide.eclipse.as.core/plugin.xml:3086]: Shouldn't we go til Java 16 as it is supported ?

**Review by jeffmaury — APPROVED** [review]
LGTM. I don't see the update to the WF 24 download URL that is used by the integration tests ?

File: as/plugins/org.jboss.ide.eclipse.as.core/plugin.xml (hunk 1 above)
Comment (robstryker) [reply as/plugins/org.jboss.ide.eclipse.as.core/plugin.xml]: Added up to java16. Thanks.

**Comment by robstryker** [comment]
Gonna merge this before I leave. So... let's have QE test it.

**Pull request merged**